### PR TITLE
Book Store - React&Redux - Milestone 5

### PR DESCRIPTION
**Pull request by 300ms**
Followed the provided design specs on Zeplin to build a UI for your app

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,49 +1,282 @@
...
+  width: 20rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
```
Comment (rindrajosia) [inline src/index.css]: - [ ] [REQUIRED] Please make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -37,13 +37,13 @@ const renderList = ({ state, handleRemoveBook, handleFilterChange }) => {
...
         <table className="fl-table">
-          <thead>
+          {/* <thead>
             <tr>
               <th>BookId</th>
               <th>Category</th>
               <th>Title</th>
             </tr>
-          </thead>
+          </thead> */}
```
Comment (rindrajosia) [inline src/containers/BooksList.js]: - [ ] [REQUIRED] Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [x] Category selector should be in the navbar.
- [x]  the space on the right and the left of the form is not equal as a result, a horizontal scroll bar appears. It will be nice if you fix it.
- [x] Please, make sure that avatar appears in your APP.
- [x] all element in the design should be added, features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/designAllElementAsPlaceholder.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/)) (Progress bar, Update progress button, etc...)


- [x] Please align the avatar to the right. Styling should be **EXACTLY** the same as the one provided in the designs ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/exactlySameAsDesign.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/))


- [x] The padding-top in the form is too big. It will be nice if you reduce it. It will be also nice if you increase a little bit the padding-bottom in the form.





_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.css [hunk 3]
```diff
@@ -1,150 +1,405 @@
...
+  background-image: url("./icons/login_icon.png");
+  background-repeat: no-repeat;
+  width: 45px;
+  height: 45px;
+  background-size: cover;
 }
 
-.book-form {
+.list-item {
   display: flex;
```
Comment (taiwo2) [inline src/index.css:141]: [REQUIRED] Please make sure that basic CSS bad practices are not used,
no repeating of the same rule

File: src/index.css (hunk 1 above)
Comment (taiwo2) [reply src/index.css]: [REQUIRED] Please make sure that basic CSS bad practices are not used.
no repeating of the same rule. kindly implement this in your styling

File: src/index.css [hunk 4]
```diff
@@ -1,150 +1,405 @@
...
+.Suzanne-Collins {
+  width: 107px;
+  height: 19px;
+  font-family: "Roboto Slab", serif;
+  font-size: 14px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
```
Comment (taiwo2) [inline src/index.css:207]: - [REQUIRED] Please make sure that basic CSS bad practices are not used.
- no repeating of the same rule. Kindly check and implement this to all your stylings

File: src/index.css [hunk 5]
```diff
@@ -1,150 +1,405 @@
...
+  width: 68px;
+  height: 39px;
+  font-family: "Montserrat", sans-serif;
+  font-size: 32px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-three);
```
Comment (taiwo2) [inline src/index.css]: [REQUIRED] Please make sure that basic CSS bad practices are not used (article for support ) (screenshot from Pathwright)
 please check your styling to implement  `no repeating of the same rule` in all applicable styles

File: src/index.css [hunk 6]
```diff
@@ -1,150 +1,405 @@
...
+  color: var(--azure);
+}
+
+.panel-bg {
+  position: relative;
+  top: 0;
+  left: 50%;
+  margin-bottom: 40px;
+  transform: translateX(-50%);
+  display: flex;
```
Comment (taiwo2) [inline src/index.css:94]: [REQUIRED] Please make sure that basic CSS bad practices are not used, thre are many `flex`. you can create class to implement it where needed

- no repeating of the same rule

**Review by taiwo2 — CHANGES_REQUESTED** [review]
Hi @300ms , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
### Higlights

- Category selector should be in the navbar. 👍🏻 
-  the space on the right and the left of the form is not equal as a result, a horizontal scroll bar appears. It will be nice if you fix it. 👍🏻 
-  Please, make sure that avatar appears in your APP. 👍🏻 
-  all element in the design should be added, features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented (screenshot from pathwright) (Progress bar, Update progress button, etc 👍🏻 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rukundoeric — APPROVED** [review]
Hi @300ms,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**